script.js

```diff
@@ -33,8 +36,52 @@ const dataTableBody = document.querySelector('#dataTable tbody');
             statusCell.classList.add(data.status ? data.status.toLowerCase() : '');
         }
 
-        document.addEventListener('DOMContentLoaded', () => {
-            if (localJsonData && localJsonData.length > 0) {
-                localJsonData.forEach(item => addTableRowToDOM(item));
+        function updateCustomerCount(data){
+            totalCustomercount.textContent = data.length
+            activeMemberCount.textContent = data.filter(customer=> customer.status==='Active').length
```

Comment:
```suggestion
            activeMemberCount.textContent = data.filter(customer=> customer.status=== ACTIVE).length
```
this ACTIVE should be a constant and hold "active". For variables status naming, we generally use lowercase